### ISSUE TITLE
trycycler reconcile: error: argument trycycler reconcile: error: argument trycycler reconcile: error: argument --min_1kbp_identity: invalid int value: '25.0'

### ISSUE THREAD

**Issue by danpal96** [rrwick/Trycycler#62]
trycycler 0.5.4

The options `--min_1kbp_identity` can't parse floats, for example, `25.0` gives the next error:
```
trycycler reconcile: error: argument trycycler reconcile: error: argument trycycler reconcile: error: argument --min_1kbp_identity: invalid int value: '25.0'
```

**Comment by danpal96** [comment]
the same with `min_read_cov` from `trycycler consensus`

**Comment by rrwick** [comment]
Fixed in aa40d3f. Thanks!
